src/flows/components/Sidebar.tsx

```diff
@@ -214,6 +214,13 @@ const Sidebar: FC = () => {
           title: 'Export to Client Library',
           menu: <ClientList />,
         },
+        {
+          title: 'Preview Subsequent',
+          action: () => {
+            event(`Querying panel ${flow.data.get(id).id} and its dependents`)
```

Comment:
wondering how relevant it is to get the id for eventing vs just knowing how often this triggered? Feels excessive